pyproject.toml

```diff
@@ -16,6 +16,7 @@ classifiers = [
 ]
 keywords = ["mediacatch", "speech to text", "uploader", "nlp"]
 dependencies = [
+    "importlib_metadata",
```

Comment:
this package is required by python3.7
from python version 3.8 we can use the built-in method: https://docs.python.org/3.8/library/importlib.metadata.html

if decided to drop our support for python 3.7 we can remove this package from dependencies list

Reply:
Let's keep support for 3.7